### PR TITLE
No owners banner in blob view displays for only-signal-rows

### LINKED ISSUE #51866: No owners info in the bottom panel if no owners and signals present
We should display the same banner showing that this file has no ownership data associated with it, if there are only signals available.

### PR DESCRIPTION
**Pull request by cbart**
Closes #51866

I decided to implement the logic in graphQL rather than in the front-end. I think it might scale better going forwards, especially after introducing assigned ownership.

## Test plan

Unit tests + visual assessment:

Screenshot 2023-05-15 at 1 54 04 PM

### CODE REVIEW

**Comment by sourcegraph-buildkite** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.16 kb) | 0.00% (+0.16 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/cb/no-owners-with-signals-only/compare-report.html;tab=live_object) for a full comparison between the commits 9917273 and fdc8ec9 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/repo/blob/own/FileOwnershipPanel.tsx [hunk 1]
```diff
@@ -67,9 +67,13 @@ export const FileOwnershipPanel: React.FunctionComponent<
         data.node.commit?.blob &&
         data.node.commit.blob.ownership.nodes.length > 0
     ) {
+        const nodes = data.node.commit.blob.ownership.nodes
```
Comment (sashaostrikov) [inline client/web/src/repo/blob/own/FileOwnershipPanel.tsx:70]: god, I love nesting

File: enterprise/cmd/frontend/internal/own/resolvers/resolvers.go [hunk 2]
```diff
@@ -92,6 +92,11 @@ func (o *ownershipReasonResolver) ToRecentViewOwnershipSignal() (res graphqlback
 	return
 }
 
+func (o *ownershipReasonResolver) makesAnOwner() bool {
+	_, ok := o.resolver.(*codeownersFileEntryResolver)
```
Comment (sashaostrikov) [inline enterprise/cmd/frontend/internal/own/resolvers/resolvers.go:96]: neat!

**Review by sashaostrikov — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**